File: 929. Unique Email Addresses/_929.py
class Solution:
    def numUniqueEmails(self, emails) -> int:
        res = set()
        for i in emails:
            local, domain = i.split('@')
            local = local.split('+')[0]
            local = ''.join(local.split('.'))

            res.add('@'.join([local, domain]))

        return len(res)

File: 929. Unique Email Addresses/929.go
package leetcode

import(
	"strings"
)

func numUniqueEmails(emails []string) int {
	res := make(map[string]bool, 0)
	for _,v := range emails{
		tmp := strings.Split(v, "@")
		local := tmp[0]
		domain := tmp[1]
		local = strings.Split(local, "+")[0]
		local = strings.Join(strings.Split(local, "."), "")
		res[strings.Join([]string{local, domain},"@")] = true
	}
	return len(res)
}